sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@
 - Renamed properties `aspect` and `opinions` to `target` and `assessments` respectively in class `MinedOpinion`.
 - Renamed classes `AspectSentiment` and `OpinionSentiment` to `TargetSentiment` and `AssessmentSentiment` respectively.
 
+**New Features**
+
+- Added parameter `pii_entity_categories` to the `recognize_pii_entities` client method.
```

Comment:
hey @abhahn I actually made a gist for this design, [here](https://gist.github.com/iscai-msft/93f0e92f2e95eb3564fd5597714e00f5) it is, sorry I should've 100% clarified this with you. js also did a [pii entity pr](https://github.com/Azure/azure-sdk-for-js/pull/14043/files) that aligns with this (idt they added an enum though)

Reply:
Ok, I've update things to align with the gist above.